test/EndToEnd-Installer.Tests/TelemetryOptOutDefaultTests.cs

```diff
@@ -1,22 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using Microsoft.DotNet.Tools.Test.Utilities;
-using DotnetCommand = Microsoft.DotNet.Tools.Test.Utilities.DotnetCommand;
-using AssemblyInfo = Microsoft.DotNet.Tools.Test.Utilities.AssemblyInfo;
-
-namespace EndToEnd.Tests
```

Comment:
Should we have this be a test vendors run ocassionally on an official build if this is not covered any longer? This seems like an important test. Or, would it be possible to have it run in a check for tests that run after the official build is done, but not in the PR build?

Reply:
@marcpopMSFT to weigh in on this since he made the test. I had discussed it with him and we decided it wasn't worth checking as part of CI runs.